octobot_trading/exchange_data/funding/channel/funding_updater.py

```diff
@@ -44,50 +43,46 @@ class FundingUpdater(funding_channel.FundingProducer):
     FUNDING_REFRESH_TIME_MIN = 0.2 * common_constants.HOURS_TO_SECONDS
     FUNDING_REFRESH_TIME_MAX = 8 * common_constants.HOURS_TO_SECONDS
 
+    def __init__(self, channel):
+        super().__init__(channel)
+
+        # create async jobs
+        self.fetch_funding_job = async_job.AsyncJob(self._funding_fetch_and_push,
+                                                    execution_interval_delay=self.FUNDING_REFRESH_TIME,
+                                                    min_execution_delay=self.FUNDING_REFRESH_TIME_MIN)
+
+    async def initialize(self) -> None:
+        """
+        Initialize data before starting fetch_funding_job
+        """
+        next_funding_times = await self._funding_fetch_and_push()
+        if not all(next_funding_time > self.FUNDING_REFRESH_TIME for next_funding_time in next_funding_times):
```

Comment:
should we also check if `next_funding_times` is not empty ?

Reply:
Right we should